cni-plugin/Dockerfile.s390x

```diff
@@ -8,11 +8,8 @@ RUN mkdir -p /opt/cni/bin
 
 FROM scratch
 
-COPY --from=ubi /licenses /licenses
```

Comment:
@caseydavenport thanks for your kind review, I guess the `license` comes from lines as following:
```
ADD licenses/ /licenses
ADD LICENSE /licenses/
```
As there is no ubi imported in this Dockerfile.s390x.
What ubi used in Dockerfile.amd64 is to copy the library.

Reply:
Yeah, you probably need to change this line:

```
FROM s390x/debian:9.8-slim
```

to this:

```
FROM s390x/debian:9.8-slim as base
```

Then change `--from=ubi` to `--from=base`

Reply:
> s390x/debian:9.8-slim

@caseydavenport honestly, I don't see `/license` folder in image `s390x/debian:9.8-slim`, here is the result
```
# docker run -it s390x/debian:9.8-slim /bin/sh
# ls /
bin   dev  home  media	opt   root  sbin  sys  usr
boot  etc  lib	 mnt	proc  run   srv   tmp  var
```
I think license comes from the source code:
```
ADD licenses/ /licenses
ADD LICENSE /licenses/
```

Reply:
Yeah, you're right it does come from the source code so we can probably just use an `ADD` directive then.

Reply:
> Yeah, you're right it does come from the source code so we can probably just use an `ADD` directive then.

Thanks @caseydavenport I guess the change is OK for you? May you help approve it?

Reply:
I think we still need a line like this:

```
COPY licenses /licenses
```

We still want to include the license files within the produced image.

Reply:
> I think we still need a line like this:
> 
> ```
> COPY licenses /licenses
> ```
> 
> We still want to include the license files within the produced image.

@caseydavenport thanks, yes, we should use COPY rather than ADD to add local license files. fixed.

Reply:
Looks like you're adding it to the UBI build stage but not the scratch stage. We need the licenses to end up in the final produced image, so the COPY line should be in the scratch build stage.

Reply:
@caseydavenport right, it's fixed in the new commit, thanks!

Reply:
@caseydavenport are you OK with the change?